phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/default/main.less

```diff
@@ -287,6 +287,18 @@
 /*******************************************************/
 /********************** miniature **********************/
 /*******************************************************/
+.listWidget {
+    .list-cell {
+        &:hover {
+          -fx-background-color: #EFEFEF;
```

Comment:
Can we introduce a global variable for this?
I guess we want to use the same color for hovering in the whole application right?